docker-entrypoint.sh

```diff
@@ -2,8 +2,9 @@
 set -e
 
 if [[ "$*" == npm*start* ]]; then
-	for dir in "$GHOST_SOURCE/content"/*/; do
-		targetDir="$GHOST_CONTENT/$(basename "$dir")"
+	baseDir="$GHOST_SOURCE/content"
+	for dir in "$baseDir"/*/ "$baseDir"/themes/*/; do
```

Comment:
I may have misunderstood how your shell script was working. I think I understand it now. Thank you for taking the time to make these adjustments.